developers.md

```diff
@@ -0,0 +1,23 @@
+# Signalen for developers and vendors
+
+The Signalen team plans to start work on making Signalen open and reusable during spring and summer 2020. We're currently working on Amsterdam's Signalen repo, with the goal of moving Signalen to a separate master repo later this year.
```

Comment:
Can we cut the second sentence. Whether we re-clone, or move the repository is not so important. We want to encourage decentralized collaboration; over time each contributing organization as well as individuals will likely have their own branches, e.g.: https://github.com/ericherman/signals